From 09622d175a5421e19dd6cf5cc65bfa3d061a8afb Mon Sep 17 00:00:00 2001
From: zengchao <chao.zeng@siemens.com>
Date: Wed, 8 Jan 2020 11:46:39 +0800
Subject: [PATCH 12/18] feat: add the flash protect for winbond flash

      as lack of the flash protect for winbond flash,
      add the flash protect for windbond

Signed-off-by: zengchao <chao.zeng@siemens.com>
---
 drivers/mtd/spi/spi-nor-core.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/spi/spi-nor-core.c b/drivers/mtd/spi/spi-nor-core.c
index 6933e99e47..b1643e3a6e 100644
--- a/drivers/mtd/spi/spi-nor-core.c
+++ b/drivers/mtd/spi/spi-nor-core.c
@@ -2437,6 +2437,7 @@ int spi_nor_scan(struct spi_nor *nor)
 	if (JEDEC_MFR(info) == SNOR_MFR_ST ||
 	    JEDEC_MFR(info) == SNOR_MFR_MICRON ||
 	    JEDEC_MFR(info) == SNOR_MFR_SST ||
+		JEDEC_MFR(info) == SNOR_MFR_WINBOND ||
 			info->flags & SPI_NOR_HAS_LOCK) {
 		nor->flash_lock = stm_lock;
 		nor->flash_unlock = stm_unlock;
-- 
2.17.1

